lib/rspec/mocks/space.rb

```diff
@@ -1,3 +1,4 @@
+require 'rspec/support/reentrant_mutex'
```

Comment:
Should be `RSpec::Support.require_rspec_support "reentrant_mutex"`

(And a blank line between this and `module RSpec` would be nice).